scripts/install/lidarr.sh

```diff
@@ -8,26 +9,28 @@
 #   changes/dates in source files. Any modifications to our software
 #   including (via compiler) GPL-licensed code must also be made available
 #   under the GPL along with build & install instructions.
+
 user=$(cut -d: -f1 < /root/.master.info)
-ip=$(ip route get 1 | sed -n 's/^.*src \([0-9.]*\) .*$/\1/p')
 distribution=$(lsb_release -is)
 version=$(lsb_release -cs)
 #shellcheck source=sources/functions/mono
 . /etc/swizzin/sources/functions/mono
 mono_repo_setup
 apt_install libmono-cil-dev libchromaprint-tools
```

Comment:
should we not cover everything per the official wiki

>sudo apt install curl mediainfo sqlite3 chromaprint
https://wiki.servarr.com/Lidarr_Installation#Debian.2FUbuntu

Reply:
Yeah those don't work
```E: Unable to locate package chromaprint```